Cargo.toml

```diff
@@ -11,6 +11,6 @@ clap = "4.4.5"
 derivative = "2.2.0"
 eyre = "0.6.8"
 jsonrpsee = "0.20.1"
-reth = { git = "https://github.com/ultrasoundmoney/reth-block-validator", branch = "changes-to-enable-validation-api-extension" }
+reth =  { git = "https://github.com/paradigmxyz/reth.git", rev = "1cccd09" }
```

Comment:
In the other pr (https://github.com/ultrasoundmoney/reth-payload-validator/pull/8) I also changed to using the reth repo directly, without fixing the commit hash.

Looking at the frequency of breaking api changes on the reth side, it might be better though to go with the fixed commit as you did here.